tests/updater/controller/v1/test_routes.py

```diff
@@ -283,9 +290,11 @@ async def test_up_to_date(async_client, test_data):
     await _poll(async_client, device.uuid, None, False)
 
 
-async def _assertLogLines(device, expected_line_count):
-    persisted_device = await Device.get(uuid=device.uuid)
-    log = persisted_device.last_log
+async def _assertLogLines(async_client, device, expected_line_count):
```

Comment:
This should be named `_assert_log_lines` to keep with Python naming conventions.

Reply:
Changed.